src/main/resources/modules/metabolic_syndrome_disease.json

```diff
@@ -154,6 +154,32 @@
       "type": "SetAttribute",
       "attribute": "prediabetes",
       "value": true,
+      "direct_transition": "Attempt_YMCA_Diabetes_Prevention_Program"
```

Comment:
Assuming the diabetes prevention submodule is included, do we really want this to be direct? What about conditional or distributed transition?